File: day-10/tries/tests/HelloTest.php
<?php
declare(strict_types=1);

use PHPUnit\Framework\TestCase;

final class HelloTest extends TestCase {
    public function testSayHello() : void {
        $this->assertEquals(
            'Hello, World!',
            Hello::sayHello('World')
        );
    }
}

?>

File: day-16/tries/src/test/kotlin/HelloTest.kt
package tries

import kotlin.test.assertEquals
import org.junit.Test

class HelloTest {
    @Test fun helloWorldReturnsPersonalizedMessage() {
        assertEquals("Hello, Molly!", sayHello("Molly"))
    }

    @Test fun helloWorldReturnsGenericMessage() {
        assertEquals("Hello, World!", sayHello())
    }
}
